doozer/doozerlib/backend/konflux_image_builder.py

```diff
@@ -190,8 +190,12 @@ async def _start_build(self, metadata: ImageMetadata, build_repo: BuildRepo, dyn
             raise ValueError(f"[{metadata.distgit_key}] Unsupported arches: {', '.join(unsupported_arches)}")
         build_platforms = [self.SUPPORTED_ARCHES[arch] for arch in arches]
         pipelineruns_api = await self._get_pipelinerun_api(dyn_client)
+
+        # Replace 'openshift-4-18-ose-installer-terraform' with '4-18-ose-installer-terraform'
+        formatted_pipelinerun_name = component_name.replace("openshift-", "", 1) if component_name.startswith("openshift-") else component_name
```

Comment:
Let's use `removeprefix` in case a component name has `openshift` in its name.
```suggestion
        formatted_pipelinerun_name = component_name.removeprefix("openshift-")
```

Or maybe just truncate the pipelinerun name if it is too long.

Reply:
>  in case a component name has openshift in its name.
yeah - my crude implementation was removing only the first occurrence.

> Let's use removeprefix
agreed - this is better